In [1]:
import numpy as np

In [2]:
a_rand=np.random.rand(1,5)

In [3]:
a_rand

array([[0.35402434, 0.04707755, 0.81779006, 0.17407815, 0.05606456]])

In [4]:
b_rand=np.random.randint(1,6,3)

In [5]:
b_rand

array([3, 5, 2])

In [6]:
c_rand=np.random.randint(2,10,3)

In [7]:
c_rand

array([3, 4, 3])

This is numbered on ordinary numbers 

In [ ]:
c_arr=np.arange(1,6,1)

In [9]:
c_arr

array([1, 2, 3, 4, 5])

This is like two and half You're seeing now the five numbers of course Because I type here 5


In [10]:
d_arr=np.linspace(50,60,5)

In [11]:
d_arr

array([50. , 52.5, 55. , 57.5, 60. ])

Now we will work on the TXT.file to R work file

In [50]:
import random 
mylist = [1,2,3]
print(mylist)


[1, 2, 3]


In [51]:
mylist.append(4)
print(mylist)


[1, 2, 3, 4]


In [52]:
mylist.append("MAT1008")

In [53]:
mylist.append([22, 33])

In [54]:
mylist.append(True)

In [55]:
mylist.extend([6,7,8,9,10])

In [56]:
print(mylist)

[1, 2, 3, 4, 'MAT1008', [22, 33], True, 6, 7, 8, 9, 10]


In [57]:
print(mylist[0])

1


In [58]:
print(mylist[-1])

10


In [59]:
print(mylist[0:5])

[1, 2, 3, 4, 'MAT1008']


In [60]:
print(mylist[:5])

[1, 2, 3, 4, 'MAT1008']


In [61]:
print(mylist[5:])

[[22, 33], True, 6, 7, 8, 9, 10]


copying

In [62]:
aList = mylist

In [63]:
aList[0] = "modifyFirstIndex"

In [64]:
print(mylist) #not copied, its modified

['modifyFirstIndex', 2, 3, 4, 'MAT1008', [22, 33], True, 6, 7, 8, 9, 10]


to have a different list

In [65]:
blist = mylist[:]

In [66]:
blist[0] = "blistFirst"


In [67]:
print(blist)


['blistFirst', 2, 3, 4, 'MAT1008', [22, 33], True, 6, 7, 8, 9, 10]


In [68]:
print("After we changed first index of list b, mylist is : \n" + str(mylist))

After we changed first index of list b, mylist is : 
['modifyFirstIndex', 2, 3, 4, 'MAT1008', [22, 33], True, 6, 7, 8, 9, 10]


In [69]:
print(mylist.index("MAT1008"))

4


In [120]:
print(mylist[4])

MAT1008


DATA ANALYSIS ON TXT DATA

In [104]:
heights = np.genfromtxt(R"C:\Users\pc\Desktop\NİLUFER 2\nilufer\NİLÜFER\ÖRNEKLEME\Mathemetical Data Analysis on Python\height.txt", delimiter=",")

In [105]:
print(len(heights))

1015


In [106]:
weights=np.genfromtxt(R"C:\Users\pc\Desktop\NİLUFER 2\nilufer\NİLÜFER\ÖRNEKLEME\Mathemetical Data Analysis on Python\weight.txt",delimiter=",")

In [107]:
print(weights)

[180. 215. 210. ... 205. 190. 195.]


In [108]:
print(len(weights))

1015


mostly when ı a give you assignment ı will ask you something like that
this weight they kept less pause ı would like the convert it kilogram ı want to use this numbers copy of course
I can convert to kilogram my heights you can give a new variable name but won't forget when you do this .You have to remember the last one if you continue This is very important
Because Their Usa especially

In [109]:
weights=weights*0.453592

In [110]:
print(weights)

[81.64656 97.52228 95.25432 ... 92.98636 86.18248 88.45044]


Same things I want to do I did the opposite 

In [111]:
heights=heights*0.0254

In [112]:
print(heights)

[1.8796 1.8796 1.8288 ... 1.905  1.905  1.8542]


In [113]:
bmi=weights/heights**2

In [114]:
print(bmi)

[23.11037639 27.60406069 28.48080465 ... 25.62295933 23.74810865
 25.72686361]


I don't know how can explain that Because I am not expert on this topic

In [115]:
print(type(bmi))

<class 'numpy.ndarray'>


In [118]:
print("Max of bmi = " + str(np.max(bmi)))  #This code directly give maximum of bmi

Max of bmi = 35.26194861031698


I can say the strong woman or man I can understand 35.26

In [119]:
print("Min of bmi = " + str(np.min(bmi)))  #This opposite (min)

Min of bmi = 19.498447103560874


In [121]:
count_bmi21orless = np.sum(bmi<21)

In [122]:
count_bmi21orless   #11 people

11

In [123]:
print("number of people having less bmi than 21 = " + str(count_bmi21orless))

number of people having less bmi than 21 = 11


I would like the put some condition to my code. I am saying can you show me like 21 or less of bmi?
 How many number(bmi) they are my result less than 21 = 11
    I want to print number of the people have bmi more than 21 and less than.
Now I'm putting 2 condition. How many number(bmi) they are my result less than 21 = 11 
You can print your string with your results of data.This is very nice.

In [124]:
bmi_25_30 = np.sum((bmi > 25) & (bmi < 30))

In [125]:
bmi_25_30

636

In [126]:
import pandas as pd

In [ ]:
grades = pd.read_csv()